apps/sms/views/conversation/js/conversation.js

```diff
@@ -476,8 +477,10 @@ var ConversationView = {
    * visible.
    */
   beforeEnter: function conv_beforeEnter(args) {
+    this.inActivityMode = !!args.isInActivity;
```

Comment:
In this WIP I made this very ugly workaround to pass ```isInActivity``` flag, but the idea for real patch is to indicate about activity mode through parameters so that view don't have to query ActivityHandler or any other thing for it.